main.py

```diff
@@ -106,10 +107,19 @@ def duplicate_nodes(sender, app_data):
 
 
 with dpg.window(
-    tag="popup_window", no_move=True, no_close=True, no_resize=True, no_collapse=True, show=False, label="Add Node"
+    tag="popup_window",
+    no_move=True,
+    no_close=True,
+    no_resize=True,
+    no_collapse=True,
+    show=False,
+    label="Options",
+    width=250,
+    height=300,
 ):
-    for module in modules[1:-1]:
-        dpg.add_button(label=module.name, tag=module.name + "_popup", callback=module.run)
+    with dpg.menu(label="Add Node", indent=5):
+        for module in modules[1:-1]:
+            dpg.add_button(label=module.name, tag=module.name + "_popup", callback=module.run, indent=3, width=200)
 
```

Comment:
we dont need a separate menu for now since we only have 1 functionality

Reply:
i know, i just added it so we dont forget it!